index.html

```diff
@@ -1,74 +1,106 @@
 <!DOCTYPE html>
-<meta charset="utf-8" />
-<meta name="viewport" content="width=device-width,initial-scale=1" />
+<html lang="en">
+  <head>
+    <title>Trust Token demo</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="origin-trial" content="AnSBDHXEt4c/eE/KhPPE6KUKx7HlXzSKVRkb9daXxroGDFkruGl+1K9DkTPpGtfUUxah1L4xMEHA1Vyf7X3mow4AAABfeyJvcmlnaW4iOiJodHRwczovL3RydXN0LXRva2VuLWRlbW8uZ2xpdGNoLm1lOjQ0MyIsImZlYXR1cmUiOiJUcnVzdFRva2VucyIsImV4cGlyeSI6MTYwMzM1NjY1MH0=">
+    <link rel="stylesheet" href="css/main.css">
+    <script src="https://cdn.jsdelivr.net/npm/cbor-js-unofficial@0.1.0-a4/cbor.min.js" defer></script>
+    <script src="public/js/issuer.js" defer></script>
+  </head>
+  <body>
+    <main>
+      <h1>Trust Token demo</h1>
 
-<meta
-  http-equiv="origin-trial"
-  content="AnSBDHXEt4c/eE/KhPPE6KUKx7HlXzSKVRkb9daXxroGDFkruGl+1K9DkTPpGtfUUxah1L4xMEHA1Vyf7X3mow4AAABfeyJvcmlnaW4iOiJodHRwczovL3RydXN0LXRva2VuLWRlbW8uZ2xpdGNoLm1lOjQ0MyIsImZlYXR1cmUiOiJUcnVzdFRva2VucyIsImV4cGlyeSI6MTYwMzM1NjY1MH0="
-/>
+      <p>This site demonstrates Trust Token <a href="https://github.com/WICG/trust-token-api#trust-token-issuance">issuance and redemption</a>, using <a href="https://boringssl.googlesource.com/boringssl/">BoringSSL</a> to create an issuance service.</p>
+      <p>View code: <a href="https://glitch.com/edit/#!/trust-tokens">Glitch</a>, <a href="https://github.com/JackJey/trust-token">GitHub</a>.</p>
+      <p>Find out more: <a href="https://web.dev/trust-tokens">Getting started with Trust Tokens</a>.</p>
+      
+      <h2>How to use this demo</h2>
+      
+      <section>
+        <p>1. Open Chrome Canary 87 (or later) from a terminal. On MacOS use the following command:</p>
+          <div id="code">
+            <svg id="copy" height="24" width="24">
+              <title>Copy code</title>
+              <path d="M0 0h24v24H0z" fill="none"/>
+              <path d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm3 4H8c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h11c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2zm0 16H8V7h11v14z"/>
+            </svg>
+            <pre id="command">
```

Comment:
commands should be generated dynamically so this should no be written statically here.

Reply:
Ah — I see.

Reply:
Reverted issuer.js to your version.

Reply:
BTW: Can we use the simpler syntax for running Chrome with flags on MacOS?

```open -a "Google Chrome Canary" --args \```